tests/unit/fixtures/browser.js

```diff
@@ -25,3 +25,12 @@ var comment = new Comment("");
 var df = new DocumentFragment();
 var range = new Range();
 var text = new Text("");
+
+// Fetch example
```

Comment:
maybe I should add that this comes from MDN https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch ?